modules/stanford_paragraph_postcard/config/install/core.entity_form_display.paragraph.stanford_postcard.default.yml

```diff
@@ -19,29 +19,35 @@ bundle: stanford_postcard
 mode: default
 content:
   field_p_postcard_body:
-    weight: 2
+    weight: 3
     settings:
       rows: 5
       placeholder: ''
     third_party_settings: {  }
     type: text_textarea
     region: content
   field_p_postcard_image:
-    weight: 1
+    weight: 2
     settings:
       progress_indicator: throbber
       preview_image_style: thumbnail
     third_party_settings: {  }
     type: image_image
     region: content
   field_p_postcard_more:
-    weight: 3
+    weight: 4
     settings:
       placeholder_url: ''
       placeholder_title: ''
     third_party_settings: {  }
     type: link_default
     region: content
+  field_p_postcard_orientation:
```

Comment:
I thought when we get the variants work included we would want the user to choose to display the postcard left or right. Maybe we wait to expose that til the variants is ready?
Also this should have the postcard component configured already:

https://github.com/SU-SWS/stanford_paragraph_types/blob/EARTH-000-postcard-orientation/modules/stanford_paragraph_postcard/config/install/core.entity_view_display.paragraph.stanford_postcard.default.yml#L21